src/JsVlcPlayer.cpp

```diff
@@ -364,9 +365,11 @@ void JsVlcPlayer::initLibvlc( const v8::Local<v8::Array>& vlcOpts )
         _libvlc = libvlc_new( static_cast<int>( libvlcOpts.size() ), libvlcOpts.data() );
     }
 
+#if defined(_DEBUG)
     if( _libvlc ) {
         libvlc_log_set(_libvlc, JsVlcPlayer::log_event_wrapper, this);
```

Comment:
Not setting this method on OSX is raising these errors:
```
[00007ff70c874568] core vout display error: Failed to change zoom
[00007ff70c874568] core vout display error: Failed to set on top
[00007ff70c874568] core vout display error: Failed to change source AR
```

So, we'll set the log method `log_event_wrapper`, but we won't do anything on release builds.